src/main/java/com/teksystems/TekcampExerciseSpring/Validator/UserValidator.java

```diff
@@ -0,0 +1,41 @@
+package com.teksystems.TekcampExerciseSpring.Validator;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import com.teksystems.TekcampExerciseSpring.Models.UserEntity;
+import com.teksystems.TekcampExerciseSpring.Services.UserService;
+
+@Component
+public class UserValidator implements Validator {
+	
+	@Autowired
+	private UserService userService;
```

Comment:
I didn't teach you `@Autowired` because constructor wiring is better in a lot of ways. Use DI by constructor